src/styles/about.scss

```diff
@@ -0,0 +1,98 @@
+.about {
+  font-size: 14px;
+  margin-bottom: 110px;
+
+  @include tablet {
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    grid-gap: 30px;
+    margin-bottom: 130px;
+  }
+
+  @include desktop {
+    max-width: 1280px;
+    grid-template-columns: repeat(12, 1fr);
+    position: relative;
+    margin-bottom: 200px;
+  }
+
+  @include big-desktop {
+    max-width: 1440px;
+  }
+
+  &__description {
+    @include tablet {
+      grid-column-start: 1;
+      grid-column-end: 4;
+    }
+
+    @include desktop {
+      padding-left: 100px;
+      grid-column-start: 1;
+      grid-column-end: 6;
+    }
+  }
+
+  &__paragraph {
+    font-size: 14px;
+    max-width: 500px;
+    margin-bottom: 30px;
+
+    @include desktop {
+      max-width: 380px;
+      // text-align: justify;
+    }
+  }
+
+  &__paragraph-2 {
+    display: none;
+
+    @include desktop {
+      display: block;
+    }
+  }
+
+  &__images {
+    @include tablet {
+      width: 100%;
+      grid-column-start: 4;
+      grid-column-end: 7;
+
+      @include desktop {
+        grid-column-start: 7;
+        grid-column-end: 13;
+
+        display: flex;
+        align-items: flex-end;
+        gap: 20px;
+      }
+    }
+  }
+
+  &__image-big {
+    @include tablet {
+      display: block;
+      width: 100%;
+      height: 0;
+      padding-bottom: 93.85%;
+      background-image: url(../images/flowers.png);
```

Comment:
It's not a good practice to use static images in CSS. Consider using the img tag in HTML and controlling its display properties with CSS for better accessibility and SEO.